logistics_project/apps/malawi/templatetags/malawi_warehouse_tags.py

```diff
@@ -3,16 +3,17 @@
 from django.core.exceptions import ObjectDoesNotExist
 from logistics.models import SupplyPoint
 from logistics.templatetags.logistics_report_tags import r_2_s_helper
+from logistics.util import config
```

Comment:
no strong feelings, was just pointing out in case you thought it was useful. i think the code is pretty inconsistent.